projects/packages/sync/src/class-defaults.php

```diff
@@ -63,7 +63,6 @@ class Defaults {
 		'image_default_link_type',
 		'infinite_scroll',
 		'infinite_scroll_google_analytics',
-		'jetpack-memberships-connected-account-id',
```

Comment:
This needs to be replaced with `JP_Memerships::$has_connected_account_option_name` value... So it can be synced...